tomviz/Operator.h

```diff
@@ -53,6 +53,7 @@ class Operator : public QObject
   virtual bool deserialize(const pugi::xml_node& ns)=0;
 
   virtual EditOperatorWidget* getEditorContents(QWidget* parent) = 0;
+  virtual bool hasCustomUI() const = 0;
```

Comment:
Yeah, this seems a little over the top - it should default to false, and let you override it if you have custom UI. I will submit a patch.